bgflow/distribution/normal.py

```diff
@@ -236,3 +236,26 @@ def dim(self):
 
     def __len__(self):
         return self._dim
+
+class MeanFreeNormalDistribution(Energy, Sampler):
+    """ Mean-free normal distribution. """
```

Comment:
Same here; just a basic API test that checks if the energy and sample methods work without errors.